bin/docker-destroy-all.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# Stop all containers
+containers=$(docker ps -a -q)
```

Comment:
Any way that could only include the containers created by this script?
It might be unwanted by developers to have their other containers removed.

Reply:
yep, definitely 👍